cloudify_aws/cloudformation/resources/stack.py

```diff
@@ -170,3 +231,66 @@ def delete(iface, resource_config, **_):
     if not name:
         name = iface.resource_id
     iface.delete({RESOURCE_NAME: name})
+
+
+@decorators.aws_resource(CloudFormationStack, RESOURCE_TYPE)
+def pull(ctx, iface, **_):
+    _pull(ctx, iface)
+
+
+def _pull(ctx, iface):
+    if not iface.exists:
+        ctx.instance.runtime_properties[STACK_RESOURCES_RUNTIME_PROP] = []
+        delete_stack_info_runtime_properties(ctx)
+        return
+    ctx.logger.debug(
+        "Detecting stack {id} drifts.".format(id=iface.resource_id))
```

Comment:
I prefer not to use `id` for a name in Python, [because it is a global](https://stackoverflow.com/questions/77552/id-is-a-bad-variable-name-in-python/28091085).